commcare_connect/reports/views.py

```diff
@@ -34,9 +34,9 @@ def _get_quarters_since_start():
 
 
 def _get_table_data_for_quarter(quarter):
-    quarter_start = date(quarter[0], quarter[1] * 3, 1)
+    quarter_start = date(quarter[0], (quarter[1] - 1) * 3 + 1, 1)
```

Comment:
nit: this can be simplified to `Q*3 - 2`

Reply:
great point! i will do that as followup so i can get this out today